src/containers/WalletTransfer/WalletTransfer.vue

```diff
@@ -898,6 +927,12 @@ export default {
     ensRule() {
       return this.selectedVerifier === ENS && this.ensError ? this.ensError : true
     },
+    bitRule() {
+      return this.selectedVerifier === BIT && this.bitError ? this.bitError : true
+    },
+    getToAddressComboboxItems() {
+      return !this.isBitMode ? this.contactList : this.multipleAddress
```

Comment:
Flip `?:` statement will be more clear and straightforward.
`this.isBitMode ? this.multipleAddress : this.contactList`